app/client/e2e/login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
```

Comment:
It might be a good idea to wrap the tests in describe blocks. You could break it up to things like `errors` `page` `redirects` for example. Could really break it up however you like as long as it makes sense. Not too important on this page since the tests are simple and there won't be very many, but it will be important on more complex pages.